jdi-light-bootstrap-tests/src/test/java/io/github/epam/bootstrap/tests/composite/section/navbar/NavbarNavsTests.java

```diff
@@ -0,0 +1,111 @@
+package io.github.epam.bootstrap.tests.composite.section.navbar;
+
+import io.github.epam.TestsInit;
+import org.testng.annotations.BeforeMethod;
+import org.testng.annotations.Test;
+
+import java.util.Arrays;
+import java.util.List;
+
+import static com.epam.jdi.light.elements.common.WindowsManager.*;
+import static com.epam.jdi.light.elements.composite.WebPage.getUrl;
+import static io.github.com.StaticSite.bsPage;
+import static io.github.com.pages.BootstrapPage.navbarNavWithDisabled;
+import static io.github.com.pages.BootstrapPage.navbarNavWithDropdown;
+import static io.github.epam.bootstrap.tests.BaseValidations.baseValidation;
+import static io.github.epam.states.States.shouldBeLoggedIn;
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.Matchers.containsString;
+import static org.hamcrest.Matchers.is;
+
+public class NavbarNavsTests extends TestsInit {
+
+    private String textNavbarBrandDisabled = "Navbar";
+    private String textNavbarBrandWithDropdown = "Navbar";
+    private List<String> textLinksnavbarNavWithDisabled = Arrays.asList("Home", "HTML 5", "Bootstrap", "Disabled");
```

Comment:
it's better to statically import the asList, so we could write:
private List<String> textLinksnavbarNavWithDisabled = asList("Home", "HTML 5", "Bootstrap", "Disabled");

(because asList is very often method and almost there is no any confusions when developer see directly asList instead of Arrays.asList so it's ok for readability)